lib/src/shared/codec_registry.dart

```diff
@@ -33,16 +32,16 @@ class CodecRegistry {
     }
   }
 
-  factory CodecRegistry.empty() {
-    return CodecRegistry(codecs: []);
-  }
+  const CodecRegistry.empty()
```

Comment:
Not actually empty anymore. I find it strange.

Reply:
Restored original behavior